src/explorer/LeetCodeNode.ts

```diff
@@ -55,4 +55,13 @@ export class LeetCodeNode {
         };
     }
 
+    public get uri(): Uri {
+        return Uri.from({
```

Comment:
Does that mean this API is not available in 1.56.0?

Reply:
Yeah, its available from `1.57.0` 😞

Reply:
Haha, so let's bump engine & dependency version to 1.57.0